content/posts/03_system_desing_interview/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "How do you conduct and evaluate a system design interview?"
+date: 2023-02-26
+tags: [
+    "interview",
+]
+draft: true,
+---
+
+1. Intro
+  * different approaches, based on the company requirements
+  * helps to evaluate and distinguish senior and staff+ candidates
+  * difficult to do well
+2. Goals of the sys design interview
+  * High level goals the same as for all interviews
+  * Would the candidate be able to do their job
+  * Which of our career levels does the candidate fit best into
+  * Would the candidate raise our bar of talents
+3. Competencies and skills to evalute
+  * prbolem understanding
+  * getting it done
+  * distributed systems knowledge
+  * technical leadership
```

Comment:
what about the `decision pragmatism` and `customer centrism`? I'd argue that all of the above points fit into the `technical leadership` one, which is super vague.

Reply:
This will be part of it. This is just high level outline. I will dive into details after we agree such outline/flow makes sense.